tests/integration/xml/mcq_with_comments_1.xml

```diff
@@ -12,8 +12,6 @@
             <tip require="elegance" width ="600" height = "800">This is something everyone has to like about this MRQ</tip>
             <tip require="beauty" width ="400" height = "600">This is something everyone has to like about beauty</tip>
             <tip reject="bugs" width = "100" height = "200">Nah, there isn\'t any!</tip>
-
-            <!--<message type="on-submit">Thank you for answering!</message> -->
```

Comment:
@bradenmacdonald this comment was intentional - there were bugs with mentoring containing XML comments. This is a XML fixture to be used in test that verified comments are just ignored. If tests fail than we probably need to update XBlock so that when it parse XML it ignore comments. Also, haven't you noticed the name of the file? :)